.devcontainer/Dockerfile

```diff
@@ -36,15 +29,6 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update -q && \
 # I added this
 RUN source "/opt/ros/${ROS_DISTRO}/setup.bash"
 
-# Spot packages
```

Comment:
Confused by these removals. This docker container is not just for CI; it's also provided to the end user in case they don't want to develop on their machine. Does removing these still let them run the ros packages? Is it getting installed during the ROSdep call downstream?

Reply:
> Does removing these still let them run the ros packages? Is it getting installed during the ROSdep call downstream?

That's correct. Packages in this repository already declare these as dependencies, with the exception of `vision_msgs`. That one is not referenced nor used by any IIRC.